README.md

```diff
@@ -8,43 +8,44 @@ If the firewall is enabled, the  ```prereq``` role is responsible to set the rig
 
 **Please note**: the port 22/tcp is used by Ansible, so make sure you have a rule for that if the firewall is enabled. 
 
-## Running with one-shot command
+## Automatic installation (using setup script)
+
+Before proceeding with the installation, please run an ```apt update``` and ``` apt upgrade ```.
 
 If three arguments are provided, the script will assume that the user wants to set up peering with a remote cluster.
-In this case, it will install Liqo locally and then configure peering with the remote cluster using the provided IP address, username, and password.
+In this case, it will install Liqo locally and then  configure peering with the remote cluster using the provided IP address, username, and password.
```

Comment:
```suggestion
In this case, it will install Liqo locally and then configure a peering with the remote cluster using the provided IP address, username, and password.
```